src/common/reducers/repositories-status.js

```diff
@@ -0,0 +1,34 @@
+import * as ActionTypes from '../actions/repository-input';
```

Comment:
Why does the repository status data exist separately from the repository data? Could this have not been part of the `repositories` reducer?